zomes/blog/code/src/main_capability.rs

```diff
@@ -16,7 +9,7 @@ zome_functions! {
                 "date_created": "now"//SystemTime::now()
                 //SystemTime::now() panics when executed in wasmi
             }
-        )) {
+        ).to_string().as_str()) {
             Ok(post_hash) => {
```

Comment:
How is it that it actually works, given these link_entries calls?

Reply:
Yes, from what Nico said it runs, it just didn't understand how it passes the tests, since link_entries isn't implemented

Reply:
Only my other comment is the "requested change". This comment is just for my understanding

Reply:
The return value of ˋhdk::link_entries() is not used. It most likely returns an error, but so far, nobody cares.. ;)